### ISSUE TITLE
Specref updating is either too infrequent or partially broken

### ISSUE THREAD

**Issue by joanmarie** [w3c/respec#1441]
I'm trying to remove all local biblio references from the ARIA repo and rely 100% on Specref, updating the latter when needed. Because I was seeing a "legacy" reference error from ReSpec, I updated Specref to [move SGML-CATALOG from legacy.json to biblio.json](https://github.com/tobie/specref/pull/430). That change was committed over three hours ago (at the time of this writing), but ReSpec is not picking up that change. If I [query Specref](http://www.specref.org/?q=SGML-CATALOG), the result is:

> [SGML-CATALOG]
> Paul Grosso. Entity Management: OASIS Technical Resolution 9401:1997 (Amendment 2 to TR 9401). 10 September 1997. URL: https://www.oasis-open.org/specs/a401.htm

But when I load the [ARIA spec](https://rawgit.com/w3c/aria/master/), I still get the errors, and the result is:

> [SGML-CATALOG]
> Paul Grosso. Entity Management: OASIS Technical Resolution 9401:1997 (Amendment 2 to TR 9401) 10 september 1007. Entity Management Subcommittee, SGML Open. URL: https://www.oasis-open.org/html/a401.htm

Note in the latter instance the month is not capitalized and the year is "1007" <insert joke about "legacy" here> rather than "1997". Correcting the date was done as part of the Specref update I did in the aforementioned pull request.

If I just need to wait longer (e.g. if the database is only updated nightly), a statement to that effect in the Respec documentation would be helpful. Otherwise, it appears that there may be a bug in fetching references.

**Comment by tobie** [comment]
You might want to clear your browser cache. ReSpec used the same API you are using. I'm getting the updated reference when looking at the ARIA spec from here.

May I kindly remind you that all of these tools are provided to you free of charge and that no one owes you anything. If you think the documentation is lacking (it wasn't in this case, but please bear with me), sending a pull request to fix it, or at the very least offering to do so is a much better strategy to get help with your problem than just complaining that the documentation isn't good enough.

**Comment by joanmarie** [comment]
@tobie: Could you please show me where I was complaining or stating an expectation of someone "owing me" something? I'm sorry for whatever language I used which made you reach that conclusion, but I'm afraid I honestly don't know what language is to blame.

In response to your reminder: I didn't know what the point of failure was, so I couldn't very well provide a patch / pull request. I normally do provide patches for those things I can fix -- like I did earlier today to update Specref. And like I've been doing in various and sundry FLOSS projects for over a decade.

FWIW.

**Comment by marcoscaceres** [comment]
@joanmarie, sorry about @tobie's tone - it came through sounding much rougher than I'm sure he intended (certainly not the Tobie I know, who is super nice!). We've had some issues in the past with people demanding things from us - I don't think @tobie meant any ill-will, and we really appreciate your contributions. 

I'm having a look to make sure nothing is broken on the ReSpec side. As Tobie said, with both Specref and ReSpec, we use very aggressive caching of Specref entries. In ReSpec, I actually store them in IndexedDB. And after the document is generated, I update the database behind the scenes by pinging the Specref server. This can sometimes result in things not showing up as quickly as one would like - but a "shift-refresh" usually fixes things.

**Comment by marcoscaceres** [comment]
Ok, so "works for me". @joanmarie, if it's still broken on your side, let me know. Worst case, I might need you to empty your IndexedDB database using the browser's Dev Tools. 

If the issue is happening in Firefox, then it might be a long standing Firefox bug that affects IndexedDB. Should be fine in all other browsers - and, as I mentioned above, a hard refresh usually fixes things.

**Comment by joanmarie** [comment]
@marcoscaceres: Thanks for checking! I *thought* I had cleared the cache, but I cannot swear to it. And it does work for me now, so I guess I hadn't cleared the cache after all. Apologies for the false alarm/report.

**Comment by tobie** [comment]
Yeah, sorry. I would have preferred to mail you privately about this, @joanmarie, but couldn't find your email address on your profile. (I now realize it's on the WAI ARIA spec but didn't think about checking there yesterday.)

We interacted a small number of times on the Specref issue tracker and on irc, and overall, the feeling I got out of it was what I expressed above (arguably not the most kindly).

I stand by that feeling (and I'm happy to discuss it over skype/voice/email), but apologize for the form and delivery.

**Comment by joanmarie** [comment]
Wow.... Well, thank you for that feedback @tobie. Yeah, I'd like to discuss this further offline. Please email me.

**Comment by tobie** [comment]
@joanmarie and I discussed this offline. Looking at our archived exchanges, it's quite clear that my feeling wasn't backed-up by any data whatsoever, but something I constructed myself (due in part to a number of unrelated interactions that @joanmarie had nothing to do with). I apologized to @joanmarie for this and I'll keep this exchange in mind the next time this feeling shows up again to make sure I don't repeat this. @joanmarie, again, I'm sorry.
